scripts/create-kind-cluster-with-SSL-passthrough.sh

```diff
@@ -65,6 +71,27 @@ while (( $# > 0 )); do
   shift
 done
 
+
+###############################################################################
+# Prefetch the images required by nginx ingress
+###############################################################################
+prefetch_images=()
+prefetch_pids=()
+if [[ "$prefetch" == "true" ]] ; then
+  curl -s "$NGINX_INGRESS_URL" | sed 's/@sha256.*$//' > "$NGINX_INGRESS_FILE"
+  IFS=' ' read -r -a prefetch_images <<< "$(cat $NGINX_INGRESS_FILE | grep 'image: ' | awk '{print $2}' | uniq | tr '\n' ' ')"
```

Comment:
FYI, you can do this with less and lower tech, because newlines are also separators in `for ...; do ...; done`. Following is a demonstration (also making the grep and uniq less fragile).

```console
bash-3.2$ prefetch_images=$(cat $NGINX_INGRESS_FILE | grep -w 'image:' | awk '{print $2}' | sort | uniq)
bash-3.2$ 
bash-3.2$ echo $prefetch_images
registry.k8s.io/ingress-nginx/controller:v1.12.0-beta.0 registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.4
bash-3.2$ echo "$prefetch_images"
registry.k8s.io/ingress-nginx/controller:v1.12.0-beta.0
registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.4
bash-3.2$ 
bash-3.2$ for img in $prefetch_images; do echo "<$img>"; done
<registry.k8s.io/ingress-nginx/controller:v1.12.0-beta.0>
<registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.4>
```